src/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <ul class="calendar__container calendar__container--month-length--28">
+        <li class="calendar__day calendar__day--start-day--wed"></li>
```

Comment:
```suggestion
    <ul class="calendar calendar--month-length--28 calendar--start-day--wed">
      <li class="calendar__day"></li>
```
Modifiers should be added to the block directly, not to its elements
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/6ea22d10-0c24-4ccf-8032-a2dae417394f)

![image](https://github.com/mate-academy/layout_calendar/assets/96409563/5499a831-6c99-46b8-a7c2-d0b57dd396b8)

Reply:
fixed